### ISSUE TITLE
char # problem

### ISSUE THREAD

**Issue by telemarket2** [data-forge-notebook/wiki#5]
display.html() function does not show the result with character #.

e.g. 
```
let data2 = `<h1 style="color:r#c90202)">Hello Data-Forge Notebook</h1>`;
display.html(data2);
```
result: --nothing--

```
let data3 = `this is a #3`;
display.html(data3);
```
result: this is a --truncated--

**Comment by ashleydavis** [comment]
This is interesting, it seems that when I fixed issue #4 for you I introduced this issue!

**Comment by ashleydavis** [comment]
Should be ok now, I've used encodeURIComponent to encode the HTML that's being passed into the iframe.

This fix will be included in release 1.9.7, should be in the next few days.
